test/functional/fixtures/ui/pages/example.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
```

Comment:
You copied the page from the `testcafe/examples` folder. I guess it's overhead.
I propose creating a simple page with 1 input or something like that.

Reply:
https://github.com/DevExpress/testcafe/pull/7372/commits/77cbcdb745de08d193c4c9cca3a81873ad168a61